### PR TITLE
Add settings.extra_classes to one column with an overlay layout

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Added settings.extra_classes to one column with an overlay layout

# Needed By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. Checkout this branch
2. Clear cache
3. Edit basic page layout
4. Add section with One Column with an Overlay layout
5. Add extra class
6. Add/create banner block and text block
7. View page and inspect to see that extra class is being applied

# Affected Projects or Products
- Cardinal Service Development

# Associated Issues and/or People
- [CSD-15](https://stanfordits.atlassian.net/browse/CSD-15)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by pookmish — APPROVED** [review]

**Comment by pookmish** [comment]
Once the tests pass, go ahead and merge it

File: templates/layouts/one-column-overlay.html.twig [hunk 1]
```diff
@@ -18,5 +18,6 @@
 {% else %}
   {% set hero_body = content.overlay %}
   {% set hero_image = content.main %}
+  {% set modifier_class = settings.extra_classes %}
   {% include "@decanter/components/hero/hero.twig" %}
```
Comment (sherakama) [inline templates/layouts/one-column-overlay.html.twig]: You'll likely want to unset the extra_classes or you will get double `class` attributes on the hero wrapper.

Comment (pookmish) [reply templates/layouts/one-column-overlay.html.twig]: ahh thats good Peer review.
@rebeccahongsf see https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/templates/components/quote/quote.html.twig#L13

Comment (rebeccahongsf) [reply templates/layouts/one-column-overlay.html.twig]: Ooo good to know! Thank you @sherakama and @pookmish 😄

**Review by sherakama — APPROVED** [review]

**Comment by sherakama** [comment]
Thank you.

**Pull request merged**